### PR TITLE
NYT-project feature branch: Model

### PR DESCRIPTION
**Pull request by diazgio**
### NYT-Article-Page 

In this project we:

- Created HTML body & CSS styles.
- Added layouts in Grid, Flex and Float.
- Added hovers to actionable links.
- Created fixed position header & footer.
- Added relevant images, videos, links and icons.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,633 @@
...
+}
+
+.most-popular h3 {
+  padding: 0 0 20px 0;
+}
+
+.most-popular-container a {
+  text-decoration: none;
+  color: black;
+}
```
Comment (bertil291utn) [inline style.css:484]: ## No repeating of the same rule[1]

Try this:
```
.color-black{
 color:black;
}
```

In your HTML elements

`<div clas="color-black">`


- [ ] Delete all `color:black;` rule you have in this file and write it the class on your HTML file

###### _[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

File: style.css [hunk 2]
```diff
@@ -0,0 +1,633 @@
...
+.nav-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  margin: 0 0 10px 0;
+}
+
+.footer-title {
+  display: flex;
+  flex-direction: row;
```
Comment (bertil291utn) [inline style.css:532]: ## No repeating of the same rule[1]

Try this:
```
.flex{
 display:flex;
}
```

In your HTML elements

`<div class="flex">`

- [ ] Delete all `display:flex;` rule you have in this file and write it the class on your HTML file

###### _[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

File: index.html [hunk 3]
```diff
@@ -0,0 +1,602 @@
...
+                        <p>
+                            Astronomers have found evidence to support the theory of inflation, 
+                            which explains how the universe expanded so uniformly and so quickly 
+                            in the instant after the Big Bang 13.8 billion years ago.
+                        </p>
+                    </div>
+                    <div class="inflation-body">
+                        <div>
+                            <img src="./graphics/inflamation-theory.JPG" alt="Inflation chart">
+                        </div>
```
Comment (bertil291utn) [inline index.html:210]: - [ ] This section shoudn't be a image, try using grid to each block.

**Review by bertil291utn — CHANGES_REQUESTED** [review]
**Status:** Changes requested 🟠  :shipit:? Not yet.

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

### Your work is amazing you master it GitHub Flow you have a professional README file and your design is neat and professional

However, in order to accomplish as the actual page, you required to do these changes, listed below: 

**Navigation bar**

- [ ] The NY Times logo is a link ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

image

##### Bottom part

- [ ] It contains links to other articles with image previews on the leftmost side ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)). Add `<a>` tag to each article.

image

##### Footer

- [ ] It should have a fixed position (it sticks to the bottom part of the screen) ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)). You don't have a footer fixed, please add it.

image

- [ ] **The Theory of Inflation** section shouldn't be screenshots, pleas use grid and  ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)) add an image then a `<figcaption>` tag for the text.

image

- [ ] Ensure **editors pick** section should be more to the right position

image

- [ ] Ensure **editors pick** section should be behind the navbar

image

### Summary

You have a nice design but are missing on the details that were required.



<hr>

Guys work in the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

**Review by expjazz — APPROVED** [review]
## Status: Project Approved :green_circle: 

### The page looks really good and you worked on all the changes requested by my colleague. Good job :clap: 

Good Luck and Happy Coding :+1: :

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Pull request merged**